src/SFA.DAS.ProviderRegistrations/Application/Commands/AddedAccountProviderCommand/AddedAccountProviderCommand.cs

```diff
@@ -13,8 +13,8 @@ public class AddedAccountProviderCommand : IRequest
 
         public AddedAccountProviderCommand(long ukprn, Guid userRef, string correlationId)
         {
-            Ukprn = Ukprn;
-            UserRef = UserRef;
+            Ukprn = ukprn;
```

Comment:
It was used but only as part of a test I think, and that test setup a mock mediator to receive the command which took an IsAny of that type so may have masked the error? 🤔 ( only because that event that comes in is from an external project )